### PR TITLE
Pods 7664

### PR DESCRIPTION
**Pull request by cyeniceri**

### CODE REVIEW

File: test/controllers/event22/AnnualAllowanceSummaryControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
```
Comment (ngmjohnstone) [inline test/controllers/event22/AnnualAllowanceSummaryControllerSpec.scala:1]: Double notice, will remove

**Review by ngmjohnstone — COMMENTED** [review]
One test to clarify, will remove double notice, otherwise looks good

File: test/pages/common/MembersDetailsPageSpec.scala [hunk 2]
```diff
@@ -32,12 +32,12 @@ class MembersDetailsPageSpec extends PageBehaviours {
...
-      beRemovable[MembersDetails](MembersDetailsPage(Event23, 0))
-    }
+//    "event23" - {
+//      beRetrievable[MembersDetails](MembersDetailsPage(Event23, 0))
+//
+//      beSettable[MembersDetails](MembersDetailsPage(Event23, 0))
+//
+//      beRemovable[MembersDetails](MembersDetailsPage(Event23, 0))
+//    }
   }
```
[the remark this answers is not in the document]
Comment (cyeniceri) [reply test/pages/common/MembersDetailsPageSpec.scala:42]: agree with Nathan

[the remark this answers is not in the document]
Comment (grantrobson) [reply test/pages/common/MembersDetailsPageSpec.scala:42]: I'd just remove - we have ticket to remove these tests in upcoming sprint anyway as they don't add value

[the remark this answers is not in the document]
Comment (ngmjohnstone) [reply test/pages/common/MembersDetailsPageSpec.scala:42]: Happy to remove

**Review by ngmjohnstone — APPROVED** [review]
Comments resolved, approved

File: app/controllers/event1/HowAddUnauthPaymentController.scala [hunk 3]
```diff
@@ -43,21 +41,18 @@ class HowAddUnauthPaymentController @Inject()(val controllerComponents: Messages
   private val eventType = EventType.Event1
 
   def onPageLoad(waypoints: Waypoints, index: Index): Action[AnyContent] = (identify andThen getData(eventType)) { implicit request =>
-    val preparedForm = request.userAnswers.flatMap(_.get(HowAddUnauthPaymentPage(index))).fold(form)(form.fill)
-    Ok(view(preparedForm, waypoints, index))
+    Ok(view(form, waypoints, index))
```
Comment (cyeniceri) [inline app/controllers/event1/HowAddUnauthPaymentController.scala:44]: why did we change this with this ticket? 
https://jira.tools.tax.service.gov.uk/browse/PODS-7664

Comment (ngmjohnstone) [reply app/controllers/event1/HowAddUnauthPaymentController.scala:44]: My understanding of this is that we didn't want to pre-fill this form and thus removed the retrieval of the selected previous answer but @grantrobson to confirm

Comment (cyeniceri) [reply app/controllers/event1/HowAddUnauthPaymentController.scala:44]: so we removed this from all the event types' entrance page ? If so, ok. I couldn't see the requirement in ticket. If we talked before, I don't remember :(

Comment (grantrobson) [reply app/controllers/event1/HowAddUnauthPaymentController.scala:44]: We don't need to save this in Mongo. It is not transformed at back end, nor is it used on any other pages. So it's just clutter.

**Pull request merged**